recipe/meta.yaml

```diff
@@ -11,13 +11,13 @@ source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
   patches:
-    - 0001-Disable-static-linking-of-freetds.patch
-    - python_38_fix.patch
-    - rm_linux_distribution.patch
+    - patches/0002-customize_compiler.patch
 
 build:
   number: 0
   script:
```

Comment:
We should skip `py<36`, see https://github.com/pymssql/pymssql/blob/master/ChangeLog.rst#general-3